src/features/donations/components/treeDonation/GiftForm.tsx

```diff
@@ -72,5 +80,5 @@ export default function GiftForm({
         </div>
       </div>
     </div>
-  ) : null;
+  ) : <></>;
```

Comment:
I put this `null` value into very many components. Do we have to replace it with  `<></>` everywhere (>70 times in our codebase)?